assets/src/edit-story/app/file/provider.js

```diff
@@ -17,31 +17,34 @@
 /**
  * External dependencies
  */
-import { useCallback, useMemo } from 'react';
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
 
-const useFontApi = (dataAdapter, { fontApi }) => {
-  const getAllFonts = useCallback(() => {
-    if (!fontApi) {
-      return Promise.resolve([]);
-    }
-
-    return dataAdapter.get(fontApi).then((data) =>
-      data.map((font) => ({
-        value: font.family,
-        name: font.family,
-        ...font,
-      }))
-    );
-  }, [dataAdapter, fontApi]);
+/**
+ * Internal dependencies
+ */
+import Context from './context';
 
-  const api = useMemo(
-    () => ({
-      getAllFonts,
-    }),
-    [getAllFonts]
+function FileProvider({ children }) {
+  const getFonts = useCallback(
```

Comment:
@swissspidy I spoke to @barklund regarding testing and he mentioned about just ignoring this file providers as it is always mocked in tests. Thoughts?

Reply:
Ignoring it where/how?

Reply:
Ignore this for test coverage tools.